msteams-platform/bots/how-to/conversations/send-and-receive-messages.md

```diff
@@ -7,6 +7,51 @@ ms.author: anclear
 ---
 # Send and receive messages
 
+<!-- Draft -->
+A bot communicates with users through message activities. Some messages are simply plain text, others contain richer content such as cards or attachments. The bot's turn handler receives messages from the user, and sends responses back. The **turn context** object provides methods for sending messages back to the user.
+Markdown is supported for most text fields, but support may vary by channel.
+
+## Send a message
+
+To send a simple text message, specify the string you want to send as the activity.
+In the bot's activity handlers, use the turn context object's `SendActivityAsync` method to send a single message response. You can also use the object's `SendActivitiesAsync` method to send multiple responses at once.
+
+[!code-csharp[Send message](https://github.com/microsoft/botbuilder-dotnet/blob/master/tests/Teams/Roster/Bots/RosterBot.cs?range=23&highlight=23)]
+
+```cs
+await turnContext.SendActivityAsync(replyActivity, cancellationToken);
+
+```
+
+## Receive a message
+
+To receive a simple text message, use the *text* property of the *activity* object.
+In the bot's activity handlers, use the following code to receive a message. 
+
+[!code-csharp[Receive message](https://github.com/microsoft/botbuilder-dotnet/blob/master/tests/Teams/Roster/Bots/RosterBot.cs?range=27-45&highlight=27)]
+
+
+```cs
+var selection = turnContext.Activity.Text;
+switch (selection)
+{
+  // switch statements
+}
+```
+
+## Additional resources
+
+- Inside the bots - [How bots work](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-basics?view=azure-bot-service-4.0&tabs=csharp#bot-structure)
```

Comment:
links to other doc repos should look like: /azure/bot-service/bot-builder-basics?view=azure-bot-service-4.0&tabs=csharp#bot-structure. At a minimum, be sure to remove the "en-us" from any URLs to allow localization to occur.